src/server/methods.rs

```diff
@@ -79,11 +79,23 @@ impl InstallationMethods {
             buf.push_str("No installation method found:\n");
             self.package.format_error(&mut buf);
             self.docker.format_error(&mut buf);
-            buf.push_str("Consider installing docker: \
-                https://docs.docker.com/get-docker/");
-            buf.push_str("Or ask for native support at \
-                https://github.com/edgedb/edgedb-cli/issues/new\
-                ?template=install-unsupported.md");
+            if cfg!(windows) {
+                buf.push_str("EdgeDB server installation on Windows \
+                    requires Docker Desktop to be installed and running. \
+                    You can download Docker Desktop for Windows here: \
+                    https://hub.docker.com/editions/community/docker-ce-desktop-windows/ \
+                    Once Docker Desktop is installed and running, restart the \
```

Comment:
"restart" is confnusing here. "perform"? "attempt"?

Reply:
Why is it confusing?  IMO it's the most appropriate verb here.  You started an install, it failed because you're missing a prerequisite, you fix the issue and then you restart the install.